.github/workflows/build_wheels.yml

```diff
@@ -0,0 +1,48 @@
+name: Build
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    name: Build wheels and sdist on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [macos-latest, ubuntu-latest, windows-latest]
```

Comment:
nice work, Ben. so we don't have any compatibility issues with windows anymore? i thought the "/path/dirpath/fname" hardcoding was causing issues there?

Reply:
i guess before we build the windows wheel, i think we should make sure the tests pass on windows first

so we dont give people false hope at this stage.

Reply:
Thanks, John. I think that could be causing issues, but I think it would probably show up in the tests. I'll work on modifying our test workflow accordingly.

Reply:
Made some modifications to the testing workflow. We're blocked by `disparity_interpolation`, but @jhonykaesemodel and I have discussed a longer term fix to simplify the build process.